packages/kbn-esql-validation-autocomplete/src/shared/helpers.ts

```diff
@@ -566,6 +566,10 @@ export function isRestartingExpression(text: string) {
   return getLastCharFromTrimmed(text) === ',';
 }
 
+export function shouldBeQuotedSource(text: string) {
```

Comment:
Are we testing this anywhere?

Reply:
Updated here [eccfb5e](https://github.com/elastic/kibana/pull/187899/commits/eccfb5e1de1cfd57681335ddf5ed806ed13fddc3)